pkg/config/config.go

```diff
@@ -154,34 +156,63 @@ type Notifications struct {
 
 // RedisConfig contains the configuration values for a redis system
 type RedisConfig struct {
-	Auth *url.Userinfo
-	URL  *url.URL
-
-	opt *redis.Options
-	cli *redis.Client
+	cli redis.UniversalClient
```

Comment:
Just to be sure: with this change, we won't leak the redis credentials in error messages?

Reply:
Not really, it is more because these fields were dead codes. They weren't used anywhere and have no real justification.